create-budget/azuredeploy.json

```diff
@@ -0,0 +1,160 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "budgetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Budget name"
+      }
+    },
+    "amount": {
+      "type": "string",
+      "metadata": {
+        "description": "The total amount of cost or usage to track with the budget"
+      }
+    },
+    "budgetCategory": {
+      "type": "string",
+      "defaultValue": "Cost",
+      "allowedValues": [
+        "Cost",
+        "Usage"
+      ],
+      "metadata": {
+        "description": "The category of the budget, whether the budget tracks cost or usage."
+      }
+    },
+    "timeGrain": {
+      "type": "string",
+      "defaultValue": "Monthly",
+      "allowedValues": [
+        "Monthly",
+        "Quarterly",
+        "Annually"
+      ],
+      "metadata": {
+        "description": "The time covered by a budget. Tracking of the amount will be reset based on the time grain."
+      }
+    },
+    "startDate": {
+      "type": "string",
+      "defaultValue": "YYYY-MM-DD",
```

Comment:
remove this since it's not a valid defaultValue

Reply:
Sure! Removed.